recipes/miniaudio/config.yml

```diff
@@ -1,13 +1,15 @@
 versions:
-  "0.11.2":
+  "0.11.6":
     folder: all
+  "0.11.2":
+    folder: header_only
   "0.10.40":
-    folder: all
+    folder: header_only
   "0.10.39":
-    folder: all
+    folder: header_only
   "0.10.35":
-    folder: all
+    folder: header_only
   "0.10.33":
-    folder: all
+    folder: header_only
   "0.10.32":
-    folder: all
+    folder: header_only
```

Comment:
Why not using the same build system for all versions? It's tedious to maintain 2 branches. This .c file is there since 0.10.32 at least.

Reply:
Didn't want to disrupt the existing recipe, I'll do that.

Reply:
There are files under `extras/` that dis/appeared in some releases. Is it OK to just try to copy them during `package` even if they do not exist in all versions ?
Or, we could just copy the whole `extras/` and not bother with that.